prototype_source/quantization_2_0_tutotial.rst

```diff
@@ -0,0 +1,595 @@
+                assert isinstance(input_act, Node)
+
+                act_qspec = get_act_qspec(quantization_config)
+                maxpool_node.meta["quantization_annotation"] = QuantizationAnnotation(  # type: ignore[union-attr]
+                    input_qspec_map={
+                        input_act: act_qspec,
+                    },
+                    _annotated=True,
+                )
+                output_node.meta["quantization_annotation"] = QuantizationAnnotation(
+                    output_qspec=act_qspec,
+                    _input_output_share_observers=True,
+                    _annotated=True,
+                )
+
+        def validate(self, model: torch.fx.GraphModule) -> None:
+            pass
+
+        @classmethod
+        def get_supported_operators(cls) -> List[OperatorConfig]:
+            return []
+
+    def get_symmetric_quantization_config():
+        act_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = \
+            HistogramObserver
+        act_quantization_spec = QuantizationSpec(
+            dtype=torch.int8,
+            quant_min=-128,
+            quant_max=127,
+            qscheme=torch.per_tensor_affine,
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=act_observer_or_fake_quant_ctr.with_args(eps=2**-12),
+        )
+
+        weight_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PerChannelMinMaxObserver
+        extra_args: Dict[str, Any] = {"eps": 2**-12}
+        weight_quantization_spec = QuantizationSpec(
+            dtype=torch.int8,
+            quant_min=-127,
+            quant_max=127,
+            qscheme=torch.per_channel_symmetric,
+            ch_axis=0,
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=weight_observer_or_fake_quant_ctr.with_args(**extra_args),
+        )
+
+        bias_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PlaceholderObserver
+        bias_quantization_spec = QuantizationSpec(
+            dtype=torch.float,
+            observer_or_fake_quant_ctr=bias_observer_or_fake_quant_ctr
+        )
+        quantization_config = QuantizationConfig(
+            act_quantization_spec, weight_quantization_spec, bias_quantization_spec
+        )
+        return quantization_config
+
+    if __name__ == "__main__":
+        example_inputs = (torch.randn(1, 3, 224, 224),)
+        m = torchvision.models.resnet18().eval()
+        m_copy = copy.deepcopy(m)
+        # program capture
+        m, guards = torchdynamo.export(
+            m,
+            *copy.deepcopy(example_inputs),
+            aten_graph=True,
+        )    
+        quantizer = BackendQuantizer()
+        operator_config = get_symmetric_quantization_config()
+        quantizer.set_global(operator_config)
+        m = prepare_pt2e_quantizer(m, quantizer)
+        after_prepare_result = m(*example_inputs)
+        m = convert_pt2e(m)
```

Comment:
Can we please add a conclusion with that sum ups what the user has learned, what else to read/run with links to extra resources if applicable.

Reply:
Good suggestions. The conclusion and further reading links are added.